src/scr.c

```diff
@@ -1782,6 +1783,95 @@ int SCR_Init()
     return SCR_FAILURE;
   }
 
+  /* coonfigure used libraries */
+  {
+    kvtree* axl_config = kvtree_new();
+    assert(axl_config);
+
+    if (kvtree_util_set_double(axl_config, AXL_KEY_CONFIG_FLUSH_ASYNC_BW,
+                               scr_flush_async_bw) != KVTREE_SUCCESS) {
+      scr_abort(-1, "Failed to set AXL config option %s @ %s:%d",
+        AXL_KEY_CONFIG_FLUSH_ASYNC_BW, __FILE__, __LINE__
+      );
+    }
+    if (kvtree_util_set_int(axl_config, AXL_KEY_CONFIG_FLUSH_ASYNC_PERCENT,
```

Comment:
scr_flush_async_percent is actually a double, not an int.

Reply:
Fixed in 19ebce1

Reply:
Thanks!